### ISSUE TITLE
form_configurable doesn't support arrays

### ISSUE THREAD

**Issue by strugee** [cantino/huginn#1547]
AFAICT, `form_configurable` doesn't support true arrays. Specifying `type :array` generates a form element that has multiple options, but where you can only select a single option.

Am I misunderstanding something?

**Comment by cantino** [comment]
Hmm, interesting question.  @dsander?

**Comment by dsander** [comment]
Yes that is correct at the moment it only supports pre-defined arrays.

**Comment by strugee** [comment]
Okay. I may go ahead and implement this for #1521, although if someone else wants to do it they'll probably be able to do it _far_ faster.
